spec/genre_spec.rb

```diff
@@ -0,0 +1,16 @@
+require_relative '../genre'
+
+describe Genre do
+  context 'It should create a Genre instance with name' do
+    genre = Genre.new('Way to the moon')
+    it 'creates new Genre with the given parameters' do
+      expect(genre).to be_an_instance_of Genre
+    end
+    #   it 'should check if it can be archived' do
+    #     expect(genre.can_be_archived?).to eq true
+    #   end
```

Comment:
- [ ] I would recommend removing the commented line to make the codebase cleaner.